src/sinks/discard.ts

```diff
@@ -11,6 +11,10 @@
  * limitations under the License.
  */
 
+/**
+ * Sync for observables that discards all values.
+ * Useful to leave at the end of a chain.
+ */
 export function discard() {
-  return new WritableStream();
+  return new WritableStream<unknown>();
```

Comment:
Shouldn't this be `any`?

Reply:
`unknown` also lets any type be passed in. It's a bit more strict than any because you can't convert from `unknown` to some other type automatically. In practice this probably won't affect how `discard` is used so I could remove this change.